src/config/tokens/core/_core-typography.scss

```diff
@@ -2,51 +2,54 @@
+$font-family-other-languages: "Rubik", "Noto Kufi Arabic", "Noto Sans JP", sans-serif;
+$text-font-family: $font-family-english-text, $font-family-other-languages;
+$title-font-family: $font-family-english-heading, $font-family-other-languages;
 
-// - Font definition shorthands per semantics atom definitions
-$font-h1: var(--font-weight-bold) var(--font-size-h1) / var(--font-line-height-h1) var(--title-font-family);
-$font-h2: var(--font-weight-bold) var(--font-size-h2) / var(--font-line-height-h2) var(--title-font-family);
-$font-h3: var(--font-weight-light) var(--font-size-h3) / var(--font-line-height-h3) var(--title-font-family);
-$font-h4: var(--font-weight-bold) var(--font-size-h4) / var(--font-line-height-h4) var(--title-font-family);
-$font-h5: var(--font-weight-bold) var(--font-size-h5) / var(--font-line-height-h5) var(--font-family);
-$font-general-label: var(--font-weight-normal) var(--font-size-general-label) / var(--font-line-height-general-label)
-  var(--font-family);
-$font-paragraph: var(--font-weight-normal) var(--font-size-paragraph) / var(--font-line-height-paragraph)
-  var(--font-family);
-$font-subtext: var(--font-weight-normal) var(--font-size-subtext) / var(--font-line-height-subtext) var(--font-family);
+// Font definition shorthands per semantics atom definitions
+$font-h1-bold: #{$font-weight-50} #{$font-size-50}/#{$font-line-height-30} #{$title-font-family};
+$font-h1-normal: #{$font-weight-30} #{$font-size-50}/#{$font-line-height-30} #{$title-font-family};
+$font-h1-light: #{$font-weight-10} #{$font-size-40}/#{$font-line-height-20} #{$title-font-family};
+$font-h2-bold: #{$font-weight-50} #{$font-size-40}/#{$font-line-height-20} #{$title-font-family};
+$font-h2-normal: #{$font-weight-30} #{$font-size-40}/#{$font-line-height-20} #{$title-font-family};
+$font-h2-light: #{$font-weight-10} #{$font-size-30}/#{$font-line-height-10} #{$title-font-family};
+$font-h3-bold: #{$font-weight-40} #{$font-size-30}/#{$font-line-height-20} #{$title-font-family};
+$font-h3-normal: #{$font-weight-30} #{$font-size-30}/#{$font-line-height-20} #{$title-font-family};
+$font-h3-light: #{$font-weight-10} #{$font-size-30}/#{$font-line-height-10} #{$title-font-family};
+$font-text1-bold: #{$font-weight-50} #{$font-size-20}/#{$font-line-height-30} #{$title-font-family};
+$font-text1-normal: #{$font-weight-30} #{$font-size-20}/#{$font-line-height-30} #{$title-font-family};
+$font-text2-bold: #{$font-weight-50} #{$font-size-10}/#{$font-line-height-20} #{$text-font-family};
+$font-text2-normal: #{$font-weight-30} #{$font-size-10}/#{$font-line-height-20} #{$text-font-family};
+
+// Letter spacing definition per semantics atom definitions
+$letter-spacing-h1-bold: -0.2%;
+$letter-spacing-h1-normal: -0.4%;
+$letter-spacing-h1-light: -1.5%;
+$letter-spacing-h2-bold: -0.1%;
+$letter-spacing-h2-normal: -0.5%;
+$letter-spacing-h2-light: -0.8%;
+$letter-spacing-h3-bold: -0.5%;
+$letter-spacing-h3-normal: -0.5%;
+$letter-spacing-h3-light: -0.5%;
```

Comment:
why do we need those as variables?
I think we should try to minimize the number of global variables which are not  and consider exposing classes that wraps together the font and its spacing